test/WebJobs.Script.Tests/WebHostResolverTests.cs

```diff
@@ -95,21 +95,6 @@ public void CreateScriptHostConfiguration_StandbyMode_ReturnsExpectedConfigurati
                 var hostManager = resolver.GetWebScriptHostManager(settings);
                 Assert.Null(hostManager.Instance?.TraceWriter);
 
-                // verify the internals of the composite writer
-                var fieldInfo = typeof(CompositeTraceWriter).GetField("_innerTraceWriters", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
@brettsam, I had to make this additional change. These assertions are no longer valid